src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php

```diff
@@ -106,12 +111,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             throw new \InvalidArgumentException(sprintf('File is not readable: "%s".', $file));
         }
 
-        if (null === $value) {
```

Comment:
If you look above at the possible ways to set `$value`,  a `null` value was only possible when using the `askHidden()` interactive prompt. So now, it should not be possible to have `null` at this point.